src/backend/tests/models/charge_types/test_level_800_traffic_crime.py

```diff
@@ -102,13 +108,13 @@ def test_dismissed_violation_is_not_type_eligible(self):
         assert charge.expungement_result.type_eligibility.reason == 'Ineligible under 137.225(5)'
 
     def test_convicted_infraction_is_not_type_eligible(self):
-        charge = ChargeFactory.create(statute='811135', level='Infraction Class B', disposition=self.convicted)
+        charge = ChargeFactory.create(statute='811.135', level='Infraction Class B', disposition=self.convicted)
```

Comment:
We should probably keep the format of the statute like before as it seems we can get both patterns in production.